services/metadactyl-clj/src/metadactyl/service/apps/de/jobs.clj

```diff
@@ -38,9 +37,11 @@
   [job]
   (try+
    (jex/submit-job (pre-process-jex-submission job))
+   ; FIXME: Should we let this error ripple up? Not catch it?
```

Comment:
Good call. The exception handling that this ticket is intending to fix are issues which occur during synchronous request/response transactions.

After this epic is completed, we can revisit strategies on how to add more context to asynchronous exceptions.